Makefile

```diff
@@ -102,6 +102,7 @@ install-linters: ## Install linters
 	${OPTS} go get -u golang.org/x/tools/cmd/goimports
 
 format: ## Formats the code. Must have goimports installed (use make install-linters).
+	${OPTS} go mod tidy && go mod vendor
```

Comment:
Why is vendoring dependencies also necessary when formatting the project?

Reply:
I'm open for opinions. Please check my reasoning here: https://github.com/SkycoinPro/skywire-service-proxy-discovery/pull/12#discussion_r416265768